api/src/tests/resource_groups.rs

```diff
@@ -37,13 +37,21 @@ async fn test_read_resoure_group() {
         ("resource_groups_secondary".to_string(), admin1.address()),
     ];
 
-    let path = PathBuf::from(std::env!("CARGO_MANIFEST_DIR"))
-        .join("../aptos-move/move-examples/resource_groups/primary");
-    publish_package(&mut context, &mut admin0, path, named_addresses.clone()).await;
-
-    let path = PathBuf::from(std::env!("CARGO_MANIFEST_DIR"))
-        .join("../aptos-move/move-examples/resource_groups/secondary");
-    publish_package(&mut context, &mut admin1, path, named_addresses.clone()).await;
+    let named_addresses_clone = named_addresses.clone();
+    let txn = futures::executor::block_on(async move {
```

Comment:
This change isn't strictly required, but it's right on the brink of a stack overflow on compilation (see #6523) so I kept it in the PR.